Doc/src/reference-charparsers.txt

```diff
@@ -1158,7 +1158,7 @@ val @many1Strings2@: Parser<string,'u> -> Parser<string,'u> -> Parser<string,'u>
 val @stringsSepBy@: Parser<string,'u> -> Parser<string,'u> -> Parser<string,'u>
 ``]
 [
-`stringsSepBy sp sep` parses *zero* or more occurrences of `sp` separated by `sep`
+`stringsSepBy p sep` parses *zero* or more occurrences of `p` separated by `sep`
```

Comment:
Note that in addition to documenting the `stringsSepBy1` parser, I also fixed a small inconsistency in the `stringsSepBy` documentation.